app/models/bulkrax/entry.rb

```diff
@@ -13,39 +15,56 @@ class Entry < ApplicationRecord
              :user,
              to: :parser
 
+    # return true or false here
     def build
       # attributes, files_dir = nil, files = [], user = nil
+      build_metadata
+      return false if collections_created? == false
       begin
-        @item = Bulkrax::ApplicationFactory.for(factory_class.to_s).new(build_metadata, parser.files_path, [], user).run
-      rescue => e
+        Bulkrax::ApplicationFactory.for(factory_class.to_s).new(self.parsed_metadata, parser.files_path, [], user).run
+      rescue StandardError => e
         status_info(e)
       else
         status_info
-        self.collection_id = @item.id if @item.is_a?(Collection)
       end
-      return @item
+      true
```

Comment:
I miss returning the item itself. that's quite useful for debugging.  however, @item is truthy, so I don't think it causes any other changes to go back to returning it